docs/source/guides/crosshairmodding.rst

```diff
@@ -0,0 +1,148 @@
+
+       RUI_CrosshairData
+       {
+           Crosshair_1                                                                     
+           {
+               "ui"                        "ui/crosshair_alternator"   
+           }                                                                               
+           Crosshair_2                                                                     
+           {
+               "ui"                        "ui/crosshair_tri"          
+           }                                                                              
+       }
+   }
+
+**To add more crosshairs add another Crosshair\_\ X following the
+formating in the script above.**
+
+**The limit for this seems to be 4 Crosshairs onscreen at once**
+
+How the script above appears:
+-----------------------------
+
+|triandaltCH|
+
+Adjust Crosshair Spread?
+------------------------
+
+| Simply add the following line below the  "ui" line
+| ``"base_spread"               "3.0"``
+| Below the "ui" line, Like this:
+
+::
+
+    {   
+       RUI_CrosshairData
+       {
+           Crosshair_1 
+           {
+               "ui"                        "ui/crosshair_alternator" //THis is the Croshair
+               "base_spread"               "3.0"   //This is a spread Multiplier, Line doesn't exist by default
+           }
+       }
+   }
+
+* This only affects the visual spread of the crosshair, not the actual bullet spread. Positive Values increase spread while negative decrease it. Be default it is based on the weapon's own stats.
+
+No Crosshair?
+-------------
+
+::
+
+   WeaponData
+   {   
+       RUI_CrosshairData
+       {
+           Crosshair_1 
+           {
+               "ui"                        "ui/crosshair_sniper_amped" //This means NO crosshair
+           }
+       }
+   }
+
+Crosshair Index:
+----------------
+
+These are the available crosshairs in-game, along with their in-game
+reference:
+
+|Crosshair examples|
+
+Crosshair images are taken from the modding guide on
+`https://noskill.gitbook.io/titanfall2/ <https://noskill.gitbook.io/titanfall2/>`__
+
+Examples
+--------
+
+|CH1| 
+
+|CH2|
+
+
+Extra Info
+^^^^^^^^^^
+
+
+* It is recommended to test this out in a private match first. Save any changes you made to the desired weapon's file and type ``reload`` in your console
+
+
+* Keep in mind that some weapons have animated or dynamic crosshairs. Weapons like the Charge Rifle, Cold Wae, Frag Grenade, etc... have especially animated crosshairs. which can cause weirdness or jank when used on other weapons or when using other crosshairs on them. 
+   * Animated weapons like the Charge rifle will work with animated crosshairs like ``ui/crosshair_titan_sniper``
+
+Thank you to ``Cpone#0001`` and ``Nixie#8251`` from the `Northstar
+Discord <https://northstar.tf/discord>`__ for helping me figure this out
+
+Any Issues? Create an issue, or message me on Discord
+``Mysterious#7899``
```

Comment:
Do you really want people to message you for any issue they might have creating a crosshair mod?